Tetris (.NET Core)/GameManager.cs

```diff
@@ -27,13 +27,13 @@ public void MainLoop()
                 switch (input)
                 {
                     case TetrisGameInput.Left:
-                        if (this.tetrisGame.CurrentFigureCol >= 1)
+                       if (this.tetrisGame.CanMoveToLeft())
                         {
                             this.tetrisGame.CurrentFigureCol--;
                         }
                         break;
-                    case TetrisGameInput.Right:
-                        if (this.tetrisGame.CurrentFigureCol < tetrisGame.TetrisColumns - this.tetrisGame.CurrentFigure.Height)
+                    case TetrisGameInput.Right:                        
```

Comment:
Remove extra spaces at the end of the line.

Reply:
Done.